fuse/dl/README.md

```diff
@@ -0,0 +1,75 @@
+# FuseMedML DL package
+
+The fuse.dl package contains core DL related modules that make heavy use of existing DL libraries. Currently, PyTorch and PyTorch Lightning are the two primary frameworks that FuseMedML is based on.
+
+## lightning
```

Comment:
We should mention that FuseMedML works great together with many libraries as well. 
All the options are available to the user. 
Writing the training loop directly with PyTorch and also using other libraries like PyTorch ignite

Reply:
done